test/parallel/test-fs-open-flags.js

```diff
@@ -29,15 +29,9 @@ const assert = require('assert');
 const fs = require('fs');
 const path = require('path');
 
-const O_APPEND = fs.constants.O_APPEND || 0;
-const O_CREAT = fs.constants.O_CREAT || 0;
-const O_EXCL = fs.constants.O_EXCL || 0;
-const O_RDONLY = fs.constants.O_RDONLY || 0;
-const O_RDWR = fs.constants.O_RDWR || 0;
-const O_SYNC = fs.constants.O_SYNC || 0;
-const O_DSYNC = fs.constants.O_DSYNC || 0;
-const O_TRUNC = fs.constants.O_TRUNC || 0;
-const O_WRONLY = fs.constants.O_WRONLY || 0;
+// 0 if not found in fs.constants
+const { O_APPEND = 0, O_CREAT = 0, O_EXCL = 0, O_RDONLY = 0, O_RDWR = 0,
```

Comment:
Can you keep these each on separate lines?

```js
const {
  O_APPEND = 0,
  O_CREAT = 0,
  ...
} = fs.constants;
```

Reply:
Ok for readability : I will change